### DISCUSSION TITLE
Creation of lists

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [standardnotes/forum#2569] [Suggestions] [answered]
Most note taking apps have the ability to create lists similar to microsoft word. That is, when the user types -, types text and presses enter, a new - appears on the next line. Pressing enter removes this newly created - and moves to the next line.

**Comment by myreli** [comment, marked as answer]
This is now available trough the [Super note type](https://www.youtube.com/watch?v=bkp6nrGqjUM&t=370s).
